README.md

```diff
@@ -1,2 +1,23 @@
-# Angular2-Scaffold
-Angular 2 Scaffold by ngParty team
+# Angular2 Scaffold
+
+## Install
+
+```bash
+$ npm install
+```
+
+## Start Development
+
+```bash
+$ npm run web:dev
+```
+
+open http://localhost:8080
```

Comment:
It just mean open manually...

yeah in Win you have to do `explorer "http://google.com"` I can possibly either use https://github.com/Jam3/open-url or add `--open` flag to the `webpack-dev-server` but non of this is good idea.